### PR TITLE
feat(moveUpload): Added the move upload or copy upload page

### PR DESCRIPTION
**Pull request by Shruti3004**
## Description

Added the move or copy page in the `organize/uploads`

## Changes

- Added the mechanism for copy and move uploads.

## How to test

visit to `/organize/uploads/move`

## Screenshots
Screenshot from 2021-07-03 08-25-10
Screenshot from 2021-07-03 08-25-24

### CODE REVIEW

File: src/api/getFolder.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const getAllFolders = () => {
   return sendRequest({
     url,
     method: "GET",
-    credentials: "include",
+    credentials: false,
```
Comment (GMishx) [inline src/api/getFolder.js:27]: An unintended change?

File: src/pages/Organize/Uploads/Move/index.js [hunk 2]
```diff
@@ -14,20 +14,196 @@
...
+      moveCopyUploadsList.map((id) => {
+        moveUpload(
+          moveCopyUploadData.folderId,
+          parseInt(id),
+          moveCopyUploadData.groupName
+        )
+          .then(() => {
+            setMessage({
+              type: "success",
+              text: "Successfully moved the upload",
```
Comment (GMishx) [inline src/pages/Organize/Uploads/Move/index.js:89]: Please change the message to indicate `202` response. Same for copy.

**Comment by GMishx** [comment]
Also, do use the `recursive=false` to reduce amount of data in response (unless it is cached in this PR).

File: src/api/getFolder.js (hunk 1 above)
Comment (Shruti3004) [reply src/api/getFolder.js]: Actually at @GMishx I was thinking that we are separately passing credentials include to every request instead we can provide it in the `sendRequest.js` file itself.

File: src/pages/Organize/Uploads/Move/index.js (hunk 2 above)
Comment (Shruti3004) [reply src/pages/Organize/Uploads/Move/index.js]: @GMishx updated, please review it.

**Comment by Shruti3004** [comment]
@GMishx updated as you suggested.

**Review by GMishx — APPROVED** [review]
Changes looks good. Tested, working as expected.

**Comment by Shruti3004** [comment]
@GMishx rebased the pr.

**Pull request merged**